simul/platform/runsimul.go

```diff
@@ -99,6 +105,10 @@ func Simulate(serverAddress, simul, monitorAddress string) error {
 			timeout *= 2
 		}
 		childrenWait.Record()
+		log.Lvl2("Broadcasting start")
+		for _, conode := range rootSC.Tree.Roster.List {
+			rootSC.Server.Send(conode, &startSimulation{})
+		}
```

Comment:
Should we not wait that all `Node` methods have been executed first before going on ?